server/routes/api/letter_versions.js

```diff
@@ -1,12 +1,13 @@
 const express = require('express')
-const { createClient } = require('../../db')
 const router = express.Router()
-const db = createClient()
+const Campaign = require('../../db/models/campaign')
+//const LetterVersion = require('../../db/models/letter-version')
 
 router.get('/:campaignId', async (req, res) => {
   const campaignId = req.params.campaignId
   try {
-    const result = await db('letter_versions').where('campaign_id', campaignId)
+    const result = await Campaign.relatedQuery('LetterVersions').for(campaignId)
+    //const result = await LetterVersion.query().where('campaign_id', campaignId)
```

Comment:
These both generate the same underlying SQL query. Which feels more readable? 🤷🏻‍♂️